src/index.html

```diff
@@ -22,4 +22,4 @@ <h1>Today's to do</h1>
 
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** **To Delete as a task:** Kindly make sure you add a feature that updates the order of `index` in the **Local Storage**. Right now if you delete a previous task from a list and once you add a new one you find 2 tasks to have the `same index values`.
![Screenshot from 2022-08-20 16-36-58](https://user-images.githubusercontent.com/83042742/185751832-74dd0fc9-4acb-40fd-a91b-c321f885a260.png)